### PR TITLE
feat(help-panel): non desktop version

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**

### CODE REVIEW

File: src/components/help-panel/non-desktop-shortcuts-tabs.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import { screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { HelpPanelSelectedTab } from "../../types";
+import { renderWithTheme } from "../../utils/testing-utils/render-with-theme";
+import { NonDesktopShortcutTabs } from "./non-desktop-shortcuts-tabs";
+
+import { useAppLayout } from "../../utils/layout";
+
+jest.mock("../../../public/images/trackpadTabMobile.svg");
```
Comment (belom88) [inline src/components/help-panel/non-desktop-shortcuts-tabs.spec.tsx:9]: I thought svgr icons are mocked globally. You probably don't need to mock them in tests.

**Review by belom88 — APPROVED** [review]

File: src/components/help-panel/non-desktop-shortcuts-tabs.spec.tsx (hunk 1 above)
Comment (dryabinin-actionengine) [reply src/components/help-panel/non-desktop-shortcuts-tabs.spec.tsx]: done

File: src/components/help-panel/non-desktop-shortcut-item.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
+import type { HelpShortcutItem } from "../../types";
+
+import styled from "styled-components";
+
+import PlayIcon from "../../../public/icons/play.svg?svgr";
```
[the remark this answers is not in the document]
Comment (dryabinin-actionengine) [reply src/components/help-panel/non-desktop-shortcut-item.tsx]: done

File: src/components/help-panel/help-panel.spec.tsx [hunk 3]
```diff
@@ -40,6 +46,6 @@ describe("Help Panel", () => {
   it("Should render Mobile Help Panel", () => {
     const onClose = jest.fn();
     renderWithTheme(<HelpPanel onClose={onClose} />);
-    expect(DesktopHelpPanelMock).not.toHaveBeenCalled();
+    expect(NonDesktopHelpPanelMock).toHaveBeenCalled();
```
[the remark this answers is not in the document]
Comment (dryabinin-actionengine) [reply src/components/help-panel/help-panel.spec.tsx:50]: done

**Pull request merged**